Sources/SafeDICore/Visitors/InstantiableVisitor.swift

```diff
@@ -151,6 +151,10 @@ public final class InstantiableVisitor: SyntaxVisitor {
         .skipChildren
     }
 
+    public override func visit(_ node: EnumDeclSyntax) -> SyntaxVisitorContinueKind {
+        .skipChildren
+    }
```

Comment:
This won't break any kind of nested types that we will want to cover in the future right?

Reply:
Eh, we'll cross that bridge if we find it. For now we explicitly do not care about any nested types, because property type resolution on nested types is _hard_.